teuthology/misc.py

```diff
@@ -1257,3 +1258,16 @@ def is_in_dict(searchkey, searchval, d):
         return True
     else:
         return searchval == val
+
+
+def sh(command):
+    log.debug(command)
+    output = ''
+    try:
+        output = subprocess.check_output(command, stderr=subprocess.STDOUT,
+                                         shell=True)
+    except subprocess.CalledProcessError as e:
+        log.error(command + " error " + str(e.output))
```

Comment:
``log.exception()`` would be useful here; it is the same as ``log.error()`` but adds the traceback for you.